packages/examples/.storybook/preview.js

```diff
@@ -3,10 +3,20 @@ import { initializeIcons } from '@uifabric/icons';
 import { configure, addParameters, addDecorator } from '@storybook/react';
 import { withInfo } from '@storybook/addon-info';
 import { withA11y } from '@storybook/addon-a11y';
+import { withPerformance } from 'storybook-addon-performance';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withThemeProvider, withStrictMode } from '@fluentui/storybook';
+import { withKeytipLayer } from './decorators';
 import * as React from 'react';
 
 addDecorator(withInfo());
 addDecorator(withA11y());
+addDecorator(withPerformance);
+addDecorator(withKnobs({ escapeHTML: false }));
+addDecorator(withThemeProvider);
+addDecorator(withStrictMode);
+addDecorator(withKeytipLayer);
+
```

Comment:
Later in this file, take note of this part which together with preview-loader.js is supposed to help with loading only stories for the current package.
```
  // This will be updated by preview-loader with the actual current package name
  const req = require.context('../src/PACKAGE_NAME', true, /\.Example\.tsx$/);
```